bundle/edu.gemini.spModel.smartgcal/src/test/java/edu/gemini/spModel/smartgcal/CalibrationMapTest.java

```diff
@@ -46,17 +46,17 @@ public void doesSupportGNIRSWildcards() {
         Assert.assertEquals(6, keys.size());
 
         // wildcard that matches all available values
-        // this also tests that obsoleted values are not taken into account (13 are defined, 2 of them are obsolete)
+        // Note that obsolete values are allowed!
         properties.setProperty(ConfigKeyGnirs.Values.SLIT_WIDTH.toString(), "*");
         keys = map.createConfig(properties);
-        Assert.assertEquals(11, keys.size());
+        Assert.assertEquals(13, keys.size());
```

Comment:
Right - the previous test was considering that 2 elements in the test set were obsolete, so it was accounting for that - now that we are allowing obsolete items, these 2 elements will be there, increasing the expected count by 2. Same case in a subsequent test.